webflow-hacks/form-hacks/1.ts

```diff
@@ -0,0 +1,40 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+    // Get the select field.
+    const interestSelectField = document.querySelector<HTMLSelectElement>('#interest-select-field');
+    // get the email-Form element.
+    const emailForm = document.querySelector<HTMLFormElement>('#email-form');
+    // get the success-text element.
+    const successText = document.querySelector<HTMLDivElement>('.insert-success-text');
+  
+    if(interestSelectField && emailForm && successText){
```

Comment:
Instead of creating nested logic, this code can become more legible by implementing an early return.

```typescript
// If any element is missing, return
if (!interestSelectField || !emailForm || !successText) return;
// Now you can keep coding without nesting everything inside an if statement
```